meilisearch-lib/src/tasks/task.rs

```diff
@@ -74,7 +86,17 @@ pub enum TaskEvent {
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
 pub struct Task {
     pub id: TaskId,
-    pub index_uid: IndexUid,
+    /// The name of the index the task is targeting. If it isn't targeting any idex (i.e Dump task)
+    /// then this is None
+    // TODO: when next forward breaking dumps, it would be a good idea to move this field inside of
+    // the TaskContent.
```

Comment:
I agree with you, could you create an issue to help us track this? By adding a persistent link to this part of the code.

Reply:
this is already done here https://github.com/meilisearch/meilisearch/issues/2430, will be done in this release